day_7/high_level.md

```diff
@@ -0,0 +1,6 @@
+High Level explanation of Ceasar Cipher activity:  
+1. Set up a way to get user input with gets.chomp.to_i to decide how the alphabet will shift  
+1. Choose a word to encode  
+1. Figure out how to turn letters or strings into integers  
+1. Figure out how to take the user input and shift the alphabet over to get new values  
```

Comment:
Nice start to the high level overview. Going into detail on the shift step is key to the caesar cipher.